src/constants/const.ts

```diff
@@ -0,0 +1,16 @@
+// FIXME: Use better way to define enum
+export const NameType = {
+  NICKNAME: 1,
+  ANONYMOUS: 2,
+  NICKNAME_ANONYMOUS: 3,
+  REALNAME: 4,
+  NICKNAME_REALNAME: 5,
+  ANONYMOUS_REALNAME: 6,
+  NICKNAME_ANONYMOUS_REALNAME: 7,
+} as const;
+
+export const ResponseStatusType = {
+  BEFORE_UPVOTES: "before_upvotes",
+  BEFORE_RESPONSE: "before_response",
+  AFTER_RESPONSE: "after_response",
+} as const;
```

Comment:
`typeof ResponseStatus`은 그냥 `string`이기 때문에 아래와 같이 type은 따로 export 해줍시다

```suggestion
export const ResponseStatus = {
  BEFORE_UPVOTES: "before_upvotes",
  BEFORE_RESPONSE: "before_response",
  AFTER_RESPONSE: "after_response",
} as const;

export type ResponseStatusType = (typeof ResponseStatus)[keyof typeof ResponseStatus];
```

Reply:
`NameType`도 동일하게 수정 부탁드립니다

Reply:
[fd40201](https://github.com/sparcs-kaist/ara-web/pull/13/commits/fd40201e8a4fcdd89b27b4677b43a2917905783a)